tests/conftest.py

```diff
@@ -97,6 +101,51 @@ def mock_uuid(mocker):
     return uuid
 
 
+@pytest.fixture
+def mock_default_metadata_version(mocker, backend_identifier):
+    mock_metadata_version = 1
+
+    # Mock `kartothek.core.utils.verify_metadata_version`
+    def patched__verify_metadata_version(metadata_version):
+        return metadata_version
+
+    mocker.patch(
+        "kartothek.core.utils.verify_metadata_version", patched__verify_metadata_version
+    )
+    mocker.patch(
+        "kartothek.io_components.metapartition.verify_metadata_version",
```

Comment:
To avoid having to patch the same function multiple times, a common practice is to define a private function which can be the mock target.
```python
# utils.py
def _verify_metadata_version(...):
    """
    This is factored out to be an easier target for mocking
    """
def verify_metadata_version(...):
    return _verify_metadata_version(*args, **kwargs)

# conftest.py
def mock_version(mocker):
    mocker.patch(
	"kartothek.core.utils._verify_metadata_version", patched__verify_metadata_version
    )
    return mock_version
```

This way you can mock `_verify_metadata_version` once and this will easily propagate to all users of the actual function

Reply:
Done